pdns/auth-main.cc

```diff
@@ -127,14 +127,15 @@ static vector<std::shared_ptr<UDPNameserver>> g_udpReceivers;
 NetmaskGroup g_proxyProtocolACL;
 size_t g_proxyProtocolMaximumSize;
 
-void mainthread();
+// auth-carbon.cc
+void carbonDumpThread();
```

Comment:
this avoids having `auth-carbon.hh`? maybe add a comment explaining that

Reply:
yes. added a comment